DopplerJobsServer/Program.cs

```diff
@@ -16,6 +17,28 @@ public static void Main(string[] args)
         public static IHostBuilder CreateHostBuilder(string[] args) =>
             Host.CreateDefaultBuilder(args)
                 .UseSerilog((hostContext, loggerConfiguration) => loggerConfiguration.ReadFrom.Configuration(hostContext.Configuration))
+                .ConfigureAppConfiguration((hostContext, configurationBuilder) =>
+                {
+                    // It is if you want to override the configuration in your
+                    // local environment, `*.Secret.*` files will not be
+                    // included in git.
+                    configurationBuilder.AddJsonFile("appsettings.Secret.json", true);
+
+                    // It is to override configuration using Docker's services.
+                    // Probably this will be the way of overriding the
+                    // configuration in our Swarm stack.
+                    configurationBuilder.AddJsonFile("/run/secrets/appsettings.Secret.json", true);
+
+                    // Configuration for Docker Windows containers.
+                    configurationBuilder.AddJsonFile("/ProgramData/docker/secrets/appsettings.Secret.json", true);
+
+                    // It is to override configuration using a different file
+                    // for each configuration entry. For example, you can create
+                    // the file `/run/secrets/Logging__LogLevel__Default` with
+                    // the content `Trace`. See:
+                    // https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-3.1#key-per-file-configuration-provider
+                    configurationBuilder.AddKeyPerFile("/run/secrets", true);
```

Comment:
What do you think about also adding this feature for key per file secrets?

```suggestion
                    configurationBuilder.AddKeyPerFile("/run/secrets", true);
                    
                    // Configuration for Docker Windows containers.
                    configurationBuilder.AddKeyPerFile("C:\\ProgramData\\Docker\\secrets", true);
```